index.js

```diff
@@ -100,7 +108,9 @@ const generateGtfs = async (hafas, createWritable, serviceArea, opt = {}) => {
 		if (routes.has(line.id)) return;
 		routes.set(line.id, {
 			route_id: line.id,
-			// todo: agency_id
+			// leave this temporary blank, see e.g.
```

Comment:
Please keep the todo here. Referencing the `agency.txt` entry from the `routes.txt` entry still needs to be done.